scripts/update_docs_version.py

```
@@ -0,0 +1,64 @@
+from pathlib import Path
+
+ROOT_DIR = Path(__file__).resolve().parent.parent
+DOCS_DIR = ROOT_DIR / "docs"
+README_FILE = ROOT_DIR / "README.md"
+PATTERN = re.compile(r"(koxudaxi/fastapi-code-generator@)(\d+\.\d+\.\d+)")
+
+
+def get_latest_release_version() -> str:
+    """Get the latest release tag from GitHub."""
+    result = subprocess.run(
+        ["gh", "release", "list", "--limit", "1", "--exclude-drafts", "--json", "tagName", "-q", ".[0].tagName"],
```

Ruff (0.15.10)

[error] Starting a process with a partial executable path

(S607)